gnome-ext/stylesheet.css

```
@@ -1,3 +1,12 @@
+/* Provider indicator label */
+.provider-indicator-label {
+    font-size: 10pt;
+    font-weight: bold;
+    color: currentColor;  /* Adapts to theme (dark on light, light on dark) */
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)